jetty9/src/main/docker/root.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_9_3.dtd">
+<Configure id="webapp" class="org.eclipse.jetty.webapp.WebAppContext">
+  <Set name="contextPath">/root</Set>
+  <Set name="war"><Property name="jetty.webapps" default="."/>/root.war</Set>
+  <Set name="extractWAR">true</Set>
+  <Set name="tempDirectory"><Property name="jetty.base" default="."/>/webapps/root-exploded</Set>
```

Comment:
@gregw Unpacking root.war by a Dockerfile does not seem to be a good solution, because Jetty explodes root.war with its own custom directory structure, which I think can change in later Jetty versions.

I was able to make Jetty unpack root.war into any directory other than `webapps/root`, but had trouble unpack into `webapps/root`. I will try your method, and if it works, I will use the `touch` method.